app/models/benefit_group.rb

```diff
@@ -544,7 +544,13 @@ def lookup_cached_cprf_for(carrier_id)
 
   # Provide the group size factor for this benefit group.
   def group_size_factor_for(plan)
-    if use_simple_employer_calculation_model?
+    if plan.carrier_profile.is_group_size?
```

Comment:
@lokelokesh I think you missed the purpose of this method and the associated employer_group_size_factor_sets

We already import sets of group size factors for each carrier hios id.

This supports different group sizes and factors for each carrier. 

Your method hard codes values that are carrier specific which is definitely not something we want to do.

If there is an actual bug it would be related to something in the selection of those rating factors or their application. The solution is not to add carrier specific values into the method